docs/package.json

```diff
@@ -14,39 +14,39 @@
   },
   "dependencies": {
     "d3": "^3.5.6",
-    "highlight.js": "^9.18.1",
-    "jest": "^24.9.0",
+    "highlight.js": "^9.2.0",
+    "jest": "^24.7.1",
     "marked": "^0.7.0",
-    "moment": "^2.26.0",
-    "prop-types": "^15.7.2",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-router": "^3.2.6"
+    "moment": "^2.22.2",
+    "prop-types": "^15.5.8",
+    "react": "^16.5.2",
+    "react-dom": "^16.5.2",
+    "react-router": "^3.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.10.1",
-    "@babel/core": "^7.10.2",
-    "@babel/plugin-proposal-class-properties": "^7.10.1",
-    "@babel/plugin-proposal-object-rest-spread": "^7.10.1",
-    "@babel/plugin-transform-modules-commonjs": "^7.10.1",
-    "@babel/plugin-transform-object-assign": "^7.10.1",
-    "@babel/polyfill": "^7.10.1",
-    "@babel/preset-env": "^7.10.2",
-    "@babel/preset-react": "^7.10.1",
+    "@babel/cli": "^7.1.0",
```

Comment:
I am a little confused by this, but it might just be me. In your package.json it says 7.1, but in the lock file it still has 7.10?

Reply:
I'm confused by this as well. I did this:
```
rm -rf node_modules && rm package-lock.json
npm install
cd docs
rm -rf node_modules && rm package-lock.json
npm install
```
And it's still like that.

Reply:
I think it is due to semantic versioning since we are using the ^. Lets just note that and move on with testing.